manifests/init.pp

```diff
@@ -331,13 +334,33 @@
     }
   }
 
-  if $pulpcore and !$pulpcore_mirror {
-    include foreman::config::apache
+  if $pulpcore {
+    if $shared_with_foreman_vhost {
+      include foreman::config::apache
+      $servername = $foreman::config::apache::servername
+      $priority = $foreman::config::apache::priority
+      $apache_http_vhost = 'foreman'
+      $apache_https_vhost = 'foreman-ssl'
+      Class['foreman::config::apache'] -> Class['pulpcore::apache']
+    } elsif $pulp and $pulp::manage_httpd {
+      $servername = $facts['networking']['fqdn']
+      $priority = '05'
+      $apache_http_vhost = 'pulp-http'
+      $apache_https_vhost = 'pulp-https'
+      Class['pulp::apache'] -> Class['pulpcore::apache']
+    } else {
```

Comment:
This else is stand-alone Pulpcore? aka a mirror?

Reply:
Yes, but without a Pulp 2 install. I haven't tested this branch yet, but I think you'll hit this if you spin up an EL8 proxy (if we provide the correct parameters).